src/adv.py

```diff
@@ -44,26 +44,32 @@
 player = Player('Hunter')
 current_room = room['outside']
 
+
 def move_area(p_inpt):
     global current_room
-    if p_inpt == 'n':
-        new_room = current_room.n_to
-        current_room = new_room
-        print(f"You are now at the {current_room.name}. {current_room.description}")
-    elif p_inpt == 's':
-        new_room = current_room.s_to
-        current_room = new_room
-        print(f"You are now at the {current_room.name}. {current_room.description}")
-    elif p_inpt == 'e':
-        new_room = current_room.e_to
-        current_room = new_room
-        print(f"You are now at the {current_room.name}. {current_room.description}")
-    elif p_inpt == 'w':
-        new_room = current_room.w_to
-        current_room = new_room
-        print(f"You are now at the {current_room.name}. {current_room.description}")
-    else:
-        print('That is not a direction!')
+       if p_inpt == 'n':
```

Comment:
I'm getting some indentation errors here. this if statement should be on the same like as global.